README.md

```diff
@@ -0,0 +1,40 @@
+# Lansing Codes website
+
+[Lansing Codes](https://www.lansing.codes) is a website of resources for coders
+and other tech enthusiasts in and around Lansing, Michigan.
+
+These resources include a calendar of tech events, a list of meetup groups,
+local beginner-friendly resources, shout-outs of our gracious sponsors, and
+other ways to get involved in the Lansing tech community.
+
+## Code of Conduct
+All participants are expected to treat others with respect and follow our [Code of Conduct](https://github.com/lansingcodes/www/blob/master/.github/CODE_OF_CONDUCT.md).
```

Comment:
What do you think about updating using the URL https://www.lansing.codes/code-of-conduct/ here and everywhere else in GitHub so we don't have to duplicate the Code of Conduct Markdown file in all the repositories?

Reply:
Yes, we will point to  https://www.lansing.codes/code-of-conduct/ everywhere in GitHub.